static/app/views/starfish/views/spans/spansTable.tsx

```diff
@@ -49,34 +35,50 @@ export type SpanDataRow = {
   'time_spent_percentage()': number;
 };
 
-export type Keys =
-  | 'span.description'
-  | 'span.op'
-  | 'span.domain'
-  | 'sps()'
-  | 'p95(span.self_time)'
-  | 'sps_percent_change()'
-  | `sum(${typeof SPAN_SELF_TIME})`
-  | 'time_spent_percentage()';
-export type TableColumnHeader = GridColumnHeader<Keys>;
+type Column = GridColumnHeader<keyof Row>;
+
+type ValidSort = Sort & {
+  field: keyof Row;
+};
+
+type Props = {
+  moduleName: ModuleName;
+  sort: ValidSort;
+  columnOrder?: Column[];
+  endpoint?: string;
+  limit?: number;
+  method?: string;
+  spanCategory?: string;
+};
+
+const {SPAN_SELF_TIME} = SpanMetricsFields;
+
+export const SORTABLE_FIELDS = new Set([
+  'p95(span.self_time)',
+  'percentile_percent_change(span.self_time, 0.95)',
```

Comment:
Should you use `SPAN_SELF_TIME` here, and in the other places in the PR too if they are referencing fields?
```suggestion
  `p95(${SPAN_SELF_TIME})`,
  `percentile_percent_change(${SPAN_SELF_TIME}, 0.95)`,
```

Reply:
Makes sense, thanks!